_articles/dual-booting.md

```diff
@@ -67,6 +67,10 @@ In your Windows install, open Control Panel and head to "Power Options" Select "
 
 Once in the BIOS change the Boot Order to boot Pop!\_OS first that way when the <kbd>Spacebar</kbd> is pressed systemd-boot will appear and then the Windows Boot Manager can be selected for booting Windows 10. Steps for accessing the BIOS and changing the Boot Order are found [here](/articles/boot-menu/).
 
+## Windows Drivers
+
+There are a few devices that will not have drivers and that includes the Thelio Io board as it is a simple pass-through for the drives when using Windows, there are also no ACPI table entries. 
```

Comment:
There are a few devices that will not have Windows driver support. That includes the Thelio I/O board. It doesn't need any Windows drivers, however, as it is a simple pass-through for the drives when using Windows. There are also no ACPI table entries.